Code/GraphMol/MolHash/catch_tests.cpp

```diff
@@ -733,19 +733,16 @@ TEST_CASE("GitHub Issue #6505") {
 
 TEST_CASE("Github Issue #6855 MakeScaffoldGeneric isotope removal") {
   SECTION("Extended Murcko") {
-    auto mol =
```

Comment:
Unrelated whitespace fixes (which made tha PR not apply cleanly for 2023_09_02 for me)